src/site/content/en/fast/measure-performance-with-the-rail-model/index.md

```diff
@@ -0,0 +1,374 @@
+---
+layout: post
+title: Measure performance with the RAIL model
+authors:
+  - addyosmani
+  - developit
+  - kaycebasques
+  - megginkearney
+description: |
+  RAIL model enables designers and developers to reliably target the performance optimization work that has the highest impact on user experience. Learn what goals and guidelines the RAIL model sets out and which tools you can use to achieve them.
+date: 2020-06-05
+---
+
+**RAIL** is a **user-centric** performance model that provides a structure for
+thinking about performance. The model breaks down the user's experience into key
+actions (for example, tap, scroll, load) and helps you define performance goals
+for each of them.
+
+RAIL stands for four distinct aspects of web app life cycle: response,
+animation, idle, and load. Users have different performance expectations for
+each of these contexts, so performance goals are defined based on the context
+and [UX research on how users perceive
+delays](https://www.nngroup.com/articles/response-times-3-important-limits/).
+
+<figure class="w-figure">
+  <img src="./rail.png" alt="The 4 parts of the RAIL performance model: response, animation, idle, and load.">
+  <figcaption class="w-figcaption">Figure 1. The 4 parts of the RAIL performance model
```

Comment:
```suggestion
  <figcaption class="w-figcaption">The 4 parts of the RAIL performance model
```
We don't need to explicitly label the figures. LMK if you saw this somewhere in the handbook or other web.dev content. My guess is that it was just copy-pasted from WebFu.

Reply:
Please remove any other occurrences of `Figure X` throughout the guide